### DISCUSSION TITLE
service not running... dependency issue

### DISCUSSION THREAD

**Discussion by dwayneforsyth** [dorssel/usbipd-win#559] [Q&A]
C:\WINDOWS\system32>usbipd server Logging:LogLevel:Default=Trace
dbug: Microsoft.Extensions.Hosting.Internal.Host[1]
      Hosting starting
dbug: Usbipd.PcapNg[1000]
      usbipd:PcapNg:Path = ''
dbug: Usbipd.Server[1000]
      2.4.1+1.Branch.master.Sha.90acf9456020ca8c6310ca62a71ee23cb6ca34ad
dbug: Usbipd.Server[1000]
      usbipd:TcpKeepAliveInterval = 500 ms
dbug: Usbipd.Server[1000]
      usbipd:TcpKeepAliveTime = 10000 ms
info: Microsoft.Hosting.Lifetime[0]
      Application started. Press Ctrl+C to shut down.
info: Microsoft.Hosting.Lifetime[0]
      Hosting environment: Production
info: Microsoft.Hosting.Lifetime[0]
      Content root path: C:\WINDOWS\system32
dbug: Microsoft.Extensions.Hosting.Internal.Host[2]
      Hosting started
info: Microsoft.Hosting.Lifetime[0]
      Application is shutting down...
dbug: Microsoft.Extensions.Hosting.Internal.Host[3]
      Hosting stopping
dbug: Microsoft.Extensions.Hosting.Internal.Host[4]
      Hosting stopped

C:\WINDOWS\system32>sc start usbipd
[SC] StartService FAILED 1068:

The dependency service or group failed to start.

C:\WINDOWS\system32>

> usbipd wsl list
BUSID  VID:PID    DEVICE                                                        STATE
3-3    04a5:2022  FlatbedScanner_13                                             Not attached
5-1    2833:0211  Rift Sensor                                                   Not attached
7-1    045e:001d  Microsoft Natural Keyboard Pro, Microsoft Natural Keyboar...  Not attached
7-2    046d:c52b  Logitech USB Input Device, USB Input Device                   Not attached
7-3    045e:0039  Microsoft USB IntelliMouse Optical                            Not attached
8-2    04d8:9009  Microchip WinUSB Device                                       Not attached
8-3    046d:081a  Logitech Webcam C260, Webcam C260                             Not attached
8-4    10c4:ea60  Silicon Labs CP210x USB to UART Bridge (COM6)                 Not attached
10-2   10c4:ea60  Silicon Labs CP210x USB to UART Bridge (COM5)                 Not attached

usbipd wsl attach --busid 10-2
usbipd: error: The VBoxUsbMon driver is currently not running; a reboot should fix that.

**Comment by dorssel** [comment]
> usbipd: error: The VBoxUsbMon driver is currently not running; a reboot should fix that.

You did try that, right? Do you have a full installation of VirtualBox installed, perhaps? If so, it may interfere with the VBoxUsbMon service from `usbipd-win`. If not, maybe the Windows EventLog gives a clue as to why the VBoxUsbMon service won't start correctly.
